app/controllers/benefits/RemoveCompanyBenefitController.scala

```diff
@@ -204,7 +204,8 @@ class RemoveCompanyBenefitController @Inject()(
           Redirect(controllers.routes.TaxAccountSummaryController.onPageLoad())
         case Right((mandatoryJourneyValues, optionalSeq)) =>
           val stopDate = {
-            val startOfTaxYear = langUtils.Dates.formatDate(TaxYear().start)
+            val startOfTaxYear = "2022"
+            //TODO:This does not compile when we add the new library in - langUtils.Dates.formatDate(TaxYear().start)
```

Comment:
Will approve when this is fixed by merging in https://github.com/hmrc/tai-frontend/pull/863